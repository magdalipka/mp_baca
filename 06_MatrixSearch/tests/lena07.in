1
3 7
0 0 1 1 4 4 4
1 2 2 4 4 4 4
2 2 4 6 6 6 6
3

