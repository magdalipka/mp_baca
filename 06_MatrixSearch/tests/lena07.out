RekPier: nie ma 3
RekOst: nie ma 3
IterPier: nie ma 3
IterOst: nie ma 3
---
